package.json

```diff
@@ -24,8 +24,9 @@
   "author": "Warrior!",
   "license": "MIT",
   "dependencies": {
-    "react": "^15.4.1",
-    "react-dom": "^15.5.4"
+    "prop-types": "^15.6.0",
+    "react": "^16.2.0",
```

Comment:
It isn't required to change the `react` version in this file. It can break some projects. What do you think?